dashboard/test/controllers/api/v1/pd/workshop_survey_foorm_submissions_controller_test.rb

```diff
@@ -27,6 +27,16 @@ class Api::V1::Pd::WorkshopSurveyFoormSubmissionsControllerTest < ::ActionContro
     assert_not_nil response_body['survey_submission_id']
   end
 
+  # this enforces a requirement which the RED team depends on
+  test 'new form submission is linked to workshop' do
+    post :create, params: @default_params
+    assert_response :created
+
+    foorm_submission_id = JSON.parse(response.body)['submission_id']
+    foorm_submission = Foorm::Submission.find(foorm_submission_id)
+    assert_equal @pd_summer_workshop.id, foorm_submission.workshop_metadata.pd_workshop_id
+  end
```

Comment:
I'm confused about this –– would this pass before your changes because metadata was already being added to these workshops? But the test ensures it won't break in the future?

Reply:
Yep, that tracks with what I was thinking 👍